usl_models/usl_models/flood_ml/dataset.py

```diff
@@ -16,6 +16,88 @@
 from usl_models.flood_ml import model
 
 
+def load_dataset(
```

Comment:
It looks like docstrings & comments are copy-pasted from `load_dataset_windowed`. They reference window-ing, which looks specific to `load_dataset_windowed`.

It would be helpful to explain in the docstrings when a caller should use `load_dataset` vs. `load_dataset_windowed`. Why do we need both?

We should also have a test for this function.

It may be easier to add this in another PR since it doesn't look like this function is used at present.

Reply:
Fixed.